src/Symfony/Bridge/Doctrine/Messenger/DoctrinePingConnectionMiddleware.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Messenger;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Middleware\MiddlewareInterface;
+use Symfony\Component\Messenger\Middleware\StackInterface;
+
+class DoctrinePingConnectionMiddleware implements MiddlewareInterface
+{
+    private $managerRegistry;
+    private $entityManagerName;
+    private $forceCloseConnection = false;
```

Comment:
You don't need this default value as it's given by the constructor.

Reply:
As you said, I think that creating 2 separated middlewares is better. Is it ok?